kumascript/macros/HTTPSidebar.ejs

```diff
@@ -84,6 +84,44 @@ var text = mdn.localStringMap({
     'Security': 'Seguridad en HTTP',
     'Authentication': 'Autenticación en HTTP'
   },
+  'fr': {
+    'HTTP': 'HTTP',
+    'HTTPGuide': 'Guide HTTP',
+    'Basics': 'Notions élémentaires de HTTP',
+    'Overview': 'Aperçu de HTTP',
+    'Evolution': 'Évolution de HTTP',
+    'ResourcesURI': 'Ressources et URI',
+    'Identifying': 'Identifier des ressources sur le Web',
+    'DataURIs': 'URI de données',
+    'MIMETypes': 'Introduction aux types MIME',
+    'ListMIMETypes': 'Types MIME usuels',
+    'WWWorNotWWW': 'Choisir entre les URL avec ou sans www',
+    'Messages': 'Messages HTTP',
+    'Session': 'Une session HTTP typique',
+    'Connection1x': 'Gestion des connexions avec HTTP/1.x',
+    'Ranges': 'Requêtes d\'intervalle HTTP',
+    'Redirects': 'Redirections HTTP',
+    'Conditionals': 'Requêtes HTTP conditionnelles',
+    'ContentNego': 'Négociation de contenu HTTP',
+    'Headers': 'En-têtes HTTP',
+    'Response_codes': 'Codes de réponse',
+    'Compression': 'Compression HTTP',
+    'Cookies': 'Cookies HTTP',
+    'Caching': 'Cache HTTP',
+    'CORS': 'Contrôle d\'accès HTTP',
+    'Resources': 'Spécifications HTTP',
+    'Feature_policy': 'Règles de fonctionnalité',
+    'Guides': 'Guides&nbsp;:',
+    'Reference': 'Références&nbsp;:',
```

Comment:
Merci déjà :)

```suggestion
    'Guides': 'Guides\xa0:',
    'Reference': 'Références\xa0:',
```

(cf. https://github.com/mdn/yari/pull/6224#discussion_r869849888)

Reply:
Thanks, I should have seen this :x